packages/@lwc/style-compiler/src/transform.ts

```diff
@@ -15,26 +15,18 @@ export interface Config {
         /** Name of the module to resolve custom properties lookup */
         resolverModule?: string;
     };
-
-    outputConfig?: {
-        /** Apply minification to the generated code */
-        minify?: boolean;
-    };
 }
 
 export function transform(src: string, id: string, config: Config = {}): { code: string } {
     if (src === '') {
         return { code: 'export default undefined' };
     }
 
-    let plugins = [postcssLwc()];
+    const postcssMinify = require('./postcss-minify-plugins').default;
```

Comment:
I would remove the `postcss-minify-plugins` completely for here.

As we discussed yesterday, we are not interested in preserving the code original formatting. I think always running the `postcss-minify-plugin` is appropriate in this case. This plugin is fairly expensive due to the fact that it is composed of many postcss plugins traversing the CSS AST independently. On top of this, its integration into the compilation flow is fairly brittle as you can see.

Instead of saying that we will "always minify the CSS", I think we should consider that the LWC compiler doesn't preserve the authored format and always produced compressed code.

If LWC users want to optimize their CSS (using `postcss-minify`) they can always preprocess the CSS prior to passing it to the LWC compiler.